app/src/subscriptions/models.py

```
@@ -0,0 +1,37 @@
+    )
+    # Config
+    receive_email = models.BooleanField(default=True)
+    receive_line = models.BooleanField(default=False)
+
+    # Timestamps
+    created_at = models.DateTimeField(auto_now_add=True)
+    updated_at = models.DateTimeField(auto_now=True)
+
+    class Meta:
+        UniqueConstraint(fields=["user", "series"], name="unique_subscription")
+        ordering = ["-created_at"]
```

Ruff (0.14.10)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)